core/src/com/po/fuck/weapons/LaserBeam.java

```diff
@@ -45,7 +46,9 @@ public void update(float delta) {
         List<Collidable> collidableList = All.collidableCollection.collides(polygon);
         for (Collidable collidable : collidableList) {
             if (collidable instanceof Entity) {
-                ((Entity) collidable).takeDamage(3 * delta);
+                if (((Entity)collidable).teamTag != this.teamTag) {
```

Comment:
I think this logic should be moved into some helper function(protected) in a class `Bullet` `tryDamage(Entity) -> boolean`